client/src/modules/authorization/components/ResetPassword.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { PureComponent } from 'react';
+import validator from 'validator';
+import { FormattedMessage } from 'react-intl';
+
+class ResetPassword extends PureComponent {
+  state = {
+    email: '',
+    tipVisible: false
+  };
+
+  handleInputChange = event => {
+    const {
+      target: { value }
+    } = event;
+
+    this.setState({ email: value });
+  };
+
+  handleEmailValidation = () => {
+    const { email } = this.state;
+
+    return validator.isEmail(email);
+  };
+
+  handleSubmit = event => {
+    event.preventDefault();
+    const isEmailCorrect = this.handleEmailValidation();
```

Comment:
do you need this `handleEmailValidation` method? It is used only once. You can just use validator here.